pkg/frontend/tf/config/default/default.go

```diff
@@ -28,9 +30,23 @@ func init() {
 	manager.Global.ProvideMuxImpl("jaeger-transform-config/default", newProvider, tfconfig.Provider.DefaultId)
 }
 
+type options struct {
+	frontendDomain     string
+	resourceTagsToLink []string
+}
+
+func (options *options) Setup(fs *pflag.FlagSet) {
+	fs.StringVar(&options.frontendDomain, "jaeger-frontend-domain", "", "jaeger frontend domain e.g. http://localhost:3000")
```

Comment:
It doesn't necessarily need to be a domain; Jaeger UI also supports non-root base path (through `--query.base-path`).
```suggestion
	fs.StringVar(&options.frontendDomain, "jaeger-frontend-base-url", "", "jaeger frontend URL prefix e.g. http://localhost:3000 or https://example.com/kelemetry, used for link generation")
```